.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
 - 5.6
+before_install:
+- rvm install 2.2.2
```

Comment:
I think `rvm` was already there as part of the environment. What's needed by Percy is Ruby 2 (installed there).